doc/speedup-batch/bench_output.txt

```diff
@@ -0,0 +1,137 @@
+Benchmark                          ,    Min(us)    ,    Avg(us)    ,    Max(us)    
+
+schnorrsig_sign                    ,    50.4       ,    50.5       ,    50.7    
+schnorrsig_verify                  ,    89.1       ,    89.2       ,    89.3    
+schnorrsig_batch_verify_1          ,   104.0       ,   104.0       ,   104.0    
```

Comment:
batch_verify_1 being slower than verify shouldn't be. Is it possible to revert to using non-batch validation logic for this case?

Reply:
The non-batch validation (`secp256k1_schnorr_verify`) logic looks something like this:
 - calc `rj` using `secp256k1_ecmult`: Rj = s*G - e*P
 - convert `rj` (gej) to `r` (ge)
 - check if the `r.x = sig[0:32]` and `r.y = even`

one schnorrsig occupies two points in the batch, and one tweak check occupies one point in the batch. If a batch contains two points, there is no guarantee that they are from a schnorrsig (R, P). It could be from two tweak checks. So, we can't use the `r.y = even` check.

Hence, I tried implementing a slightly modified `schnorr_verify` logic:
 - calc `neg_rj` using `secp256k1_ecmult`: neg_Rj = -s*G + batch.scalar[1]*batch.point[1]
 - check if `neg_rj + batch.point[0] == inf`

This gives somewhat better benchmarks than before:
```bash
Benchmark                          ,    Min(us)    ,    Avg(us)    ,    Max(us)    

schnorrsig_sign                    ,    49.1       ,    50.1       ,    53.4    
schnorrsig_verify                  ,    86.6       ,    87.2       ,    88.4    
schnorrsig_batch_verify_1          ,    94.7       ,    95.0       ,    95.2 
```
But `schnorrsig_batch_verify_1` is still slower than `schnorrsig_vrerify`.